.travis.yml

```diff
@@ -17,16 +17,16 @@ addons:
     - emboss
 
 before_install:
-- git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-test.git
-- git clone --branch master --depth 1 https://github.com/Ensembl/ensembl.git
+- git clone --branch release/96 --depth 1 https://github.com/Ensembl/ensembl-test.git
+- git clone --branch release/96 --depth 1 https://github.com/Ensembl/ensembl.git
 - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-hive.git
-- git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-compara.git
+- git clone --branch release/96 --depth 1 https://github.com/Ensembl/ensembl-compara.git
 - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-datacheck.git
-- git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-variation.git
+- git clone --branch release/96 --depth 1 https://github.com/Ensembl/ensembl-variation.git
 - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-orm.git
 - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-taxonomy.git
-- git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-metadata.git
-- git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-funcgen.git
+- git clone --branch release/96 --depth 1 https://github.com/Ensembl/ensembl-metadata.git
+- git clone --branch release/96 --depth 1 https://github.com/Ensembl/ensembl-funcgen.git
 - git clone --branch develop --depth 1 https://github.com/samtools/htslib.git
```

Comment:
Can you take advantage of this to point to a more stable branch for htslib? (like in production_db_rework branch)

Reply:
The point of this branch was to quickly fix the Travis problem... but I can make this change too if you want...